basics/didResolve.ts

```diff
@@ -0,0 +1,25 @@
+import { DidResolver } from '@kiltprotocol/sdk-js'
+import { Did, DidDocument } from '@kiltprotocol/types'
+
+export async function didResolve(did: Did): Promise<DidDocument | null> {
```

Comment:
Can you explain the purpose of this function? What's the need or reason for wrapping the DidResolver? If you think the resolver is complex and want a recipe to simplify it for a user, I'd suggest raising an error if the resolution metadata has an error property, and returning the DID document otherwise. When there's no error, there will always be a document.

Reply:
No, It is not for that purpose. It is to run this for the tests. I mean, in theory I could call it directly in the test files. However, I am not entirely sure that also is a nice way of testing it.

Reply:
ok I understand